app/views/_includes/applications/work-history.njk

```diff
@@ -1,25 +1,81 @@
-{% set workHistory = application.workHistory
-  | toArray
-  | sort(attribute="startDate")
+{% set workHistoryItems = application.workHistory.items
+  | sort(attribute="startDate") | reverse
 %}
-{% if workHistory.length >= 1 %}
+
+{% if workHistoryItems.length >= 1 %}
```

Comment:
You don't need the operator here, so can remove`>= 1`

Reply:
I know but I think it's clearer to read for people who don't know that 0 is falsy